pkg/define/config.go

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package define
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/sealerio/sealer/common"
+	osi "github.com/sealerio/sealer/utils/os"
+)
+
+type SealerConfig struct {
```

Comment:
i agree with you that we need a config file to configure the behavior of the sealer, such as build, run.  so, could we make a discuss together about the format of this file ? @Starnop.